bazel/foreign_cc/qatzstd.patch

```diff
@@ -0,0 +1,85 @@
+diff --git a/src/Makefile b/src/Makefile
+index 1abf10d..a0c7e9a 100644
+--- a/src/Makefile
++++ b/src/Makefile
+@@ -41,6 +41,7 @@ LIBDIR     ?= $(INSTALLDIR)/lib
+ INCLUDEDIR ?= $(INSTALLDIR)/include
+ 
+ CP ?= cp
++MKDIR ?= mkdir
+ 
+ ENABLE_USDM_DRV ?= 0
+ ifneq ($(ICP_ROOT), )
+@@ -55,10 +56,8 @@ ifneq ($(ICP_ROOT), )
+ else
+ 	QATFLAGS = -DINTREE
+ 	LDFLAGS  = -lqat
+-	ifneq ($(ENABLE_USDM_DRV), 0)
+-		QATFLAGS += -DENABLE_USDM_DRV
+-		LDFLAGS	 += -lusdm
+-	endif
++	QATFLAGS += -DENABLE_USDM_DRV
```

Comment:
This PR in general is LGTM except for this patch. Is there a way you can make changes upstream so Envoy builds patch-less? Thanks. This really helps with maintenance.

Reply:
Hi, we have reduced most of the patch, and the remaining code in the patch will be updated in next release, can you help to continue reviewing? Thanks!

Reply:
OK, thanks, I think this is fine, in particular if this is gone in the next release. Appreciate the upstream work here!